jetbrains-core/src/software/aws/toolkits/jetbrains/ui/resources/PrefixBasedCompletionProvider.kt

```diff
@@ -0,0 +1,8 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.ui.resources
+
+import com.intellij.util.TextFieldCompletionProviderDumbAware
+
+abstract class PrefixBasedCompletionProvider(caseInsensitivity: Boolean) : TextFieldCompletionProviderDumbAware(caseInsensitivity)
```

Comment:
Where would this be used?

Reply:
Whoops, WIP thing but I did a `git add jetbrains-core/` so it added it.